tests/ui/consts/const-try.stderr

```diff
@@ -2,7 +2,7 @@ error: const `impl` for trait `FromResidual` which is not marked with `#[const_t
   --> $DIR/const-try.rs:15:12
    |
 LL | impl const FromResidual<Error> for TryMe {
-   |            ^^^^^^^^^^^^^^^^^^^
+   |            ^^^^^^^^^^^^^^^^^^^ not marked with `#[const_trait]`
    |
    = note: marking a trait with `#[const_trait]` ensures all default method bodies are `const`
```

Comment:
Unrelated, but this note is kinda nonsense?

Reply:
It's an untouched note since at least 2022 (and before that it was moved from somewhere else), so it'll be hard to figure out what they wanted to say then :)

I think that with the current context we can remove this?

Reply:
sure, or it could just be guaranteed to only trigger within default methods, but that might be difficult to fix, and i'd rather not see hacks to do that.